ospf6d/ospf6_message.c

```diff
@@ -576,7 +576,7 @@ static void ospf6_dbdesc_recv_slave(struct ospf6_header *oh,
 	/* else fall through to ExStart */
 	/* fallthru */
 	case OSPF6_NEIGHBOR_EXSTART:
-		/* If the neighbor is Master, act as Slave. Schedule
+		/* If the neighbor is Master, act as Secondary. Schedule
```

Comment:
Please don't change half of a pair.  "master" and "slave" forms a connected pair;  if we change "slave" to "secondary" we should change "master" to "primary" too.

(Not sure what the paired word for "member" would be...)

Reply:
Agree @eqvinox, I was waiting on feedback on whether we want to change "master" too across the sources and what possible alternatives. Will mark this and similar use cases to be "primary"